pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test: TestTenantAutoUpgrade failed

pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test.TestTenantAutoUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137500?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12137500?buildTab=artifacts#/) on master @ [5af56c0ecd701d452c19fb89908012294b5336aa](https://github.com/cockroachdb/cockroach/commits/5af56c0ecd701d452c19fb89908012294b5336aa):


```
=== RUN   TestTenantAutoUpgrade
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade1757179939
    test_log_scope.go:81: use -show-logs to present logs inline
    tenant_upgrade_test.go:247: failed to receive the right auto upgrade status after 20 seconds
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade1757179939
--- FAIL: TestTenantAutoUpgrade (42.55s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/multi-tenant
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantAutoUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-32262

pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test.TestTenantAutoUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149242?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12149242?buildTab=artifacts#/) on master @ [619aa6edad0269f75d12cb7d4b88b52f09825c7b](https://github.com/cockroachdb/cockroach/commits/619aa6edad0269f75d12cb7d4b88b52f09825c7b):


```
=== RUN   TestTenantAutoUpgrade
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade2658169137
    test_log_scope.go:81: use -show-logs to present logs inline
    tenant_upgrade_test.go:247: failed to receive the right auto upgrade status after 20 seconds
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade2658169137
--- FAIL: TestTenantAutoUpgrade (43.11s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112159 pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test: TestTenantAutoUpgrade failed [C-test-failure O-robot T-multitenant branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantAutoUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test.TestTenantAutoUpgrade [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177118?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12177118?buildTab=artifacts#/) on master @ [f2bfa2a140fa302e79c9d31b84b944a6ac5e98ce](https://github.com/cockroachdb/cockroach/commits/f2bfa2a140fa302e79c9d31b84b944a6ac5e98ce):


```
=== RUN   TestTenantAutoUpgrade
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade1059646552
    test_log_scope.go:81: use -show-logs to present logs inline
    tenant_upgrade_test.go:247: failed to receive the right auto upgrade status after 20 seconds
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/3a7f184254b17b9b5453b7560bf3e9f2/logTestTenantAutoUpgrade1059646552
--- FAIL: TestTenantAutoUpgrade (43.02s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112159 pkg/ccl/kvccl/kvtenantccl/upgradeccl/upgradeccl_test: TestTenantAutoUpgrade failed [C-test-failure O-robot T-multitenant branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantAutoUpgrade.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/kvccl/kvtenantccl/upgradeccl/tenant_upgrade_test.go
+++ b/pkg/ccl/kvccl/kvtenantccl/upgradeccl/tenant_upgrade_test.go
@@ -233,7 +233,7 @@ func TestTenantAutoUpgrade(t *testing.T) {
 		UpgradeTo roachpb.Version
 	}
 	succeedsSoon := 20 * time.Second
-	if util.RaceEnabled {
+	if util.RaceEnabled || skip.Stress() {
 		succeedsSoon = 60 * time.Second
 	}
 	// Wait for auto upgrade status to be received by the testing knob.

